### PR TITLE
[15.0][IMP] brand: test performance improvement

### PR DESCRIPTION
**Pull request by josep-tecnativa**
CC @Tecnativa TT45458

### CODE REVIEW

**Comment by josep-tecnativa** [comment]
ping @pedrobaeza

File: product_brand/tests/test_product_brand.py [hunk 1]
```diff
@@ -5,6 +5,20 @@
 
 
 class TestProductBrand(CommonCase):
+    @classmethod
+    def setUpClass(cls):
```
Comment (pedrobaeza) [inline product_brand/tests/test_product_brand.py]: You have already put this in the base test. No need to put it again.

Comment (josep-tecnativa) [reply product_brand/tests/test_product_brand.py]: Okay, thanks for the correction. I have already made the changes

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**